### ISSUE TITLE
Render the JSON schema on the site

### ISSUE THREAD

**Issue by woodruffw** [C2SP/x509-limbo#112]
This is purely a "nice to have": we could use [json-schema-for-humans](https://github.com/coveooss/json-schema-for-humans) to convert `limbo-schema.json` into Markdown and embed it as a page.

**Comment by woodruffw** [comment]
Played around with this a bit, and there are some blockers: `json-schema-for-humans` doesn't accept Draft 4, which is the JSON Schema format that Pydantic v1 emits. So we'll probably need to unblock #56 first.

**Comment by woodruffw** [comment]
WIP: #143.
